LookupAnything/i18n/zh.json

```diff
@@ -130,24 +130,23 @@
     /*********
     ** Conditions & context tags
     *********/
-    // TODO
-    "condition-or-context-tag.negate": "除了{{value}}",
+    "condition-or-context-tag.negate": "!{{value}}",
 
-    "condition.item-type.input": "input item",
-    "condition.item-type.target": "target item",
+    "condition.item-type.input": "输入物品",
+    "condition.item-type.target": "产物",
 
-    "condition.item-context-tag": "{{item}} has context tag {{tags}}",
-    "condition.item-context-tags": "{{item}} has context tags {{tags}}",
+    "condition.item-context-tag": "{{item}}有{{tags}}标签",
+    "condition.item-context-tags": "{{item}}有{{tags}}标签",
     "condition.item-context-tag.value": "'{{tag}}'", // the format of each context tag in the list (each context tag isn't translated)
 
-    "condition.item-edibility.edible": "{{item}} is edible",
-    "condition.item-edibility.min": "{{item}} has edibility of {{min}} or more",
-    "condition.item-edibility.range": "{{item}} has edibility of {{min}} to {{max}}",
+    "condition.item-edibility.edible": "{{item}}是可食用的",
+    "condition.item-edibility.min": "{{item}}的可食用性大于等于{{min}}",
+    "condition.item-edibility.range": "{{item}}的可食用性为{{min}}到{{max}}",
 
     "context-tag.bone": "骨头（任意）",
     "context-tag.egg": "鸡蛋（任意）",
     "context-tag.large-egg": "大鸡蛋（任意）",
-    "context-tag.preserved-item": "preserved {{name}}", // TODO
+    "context-tag.preserved-item": "{{name}}",

```

Comment:
Hi! This text is shown when a recipe needs a preserved item. For example, Blueberry Wine is 'preserved' blueberries. It will probably be confusing to show Blueberry in that case, since you can't use blueberries in the recipe directly.